frontend/src/assets/styles/_extra.scss

```diff
@@ -318,3 +318,14 @@ div.messageBodyLinks {
     vertical-align: middle;
   }
 }
+
+.comment-textarea {
+  box-sizing: border-box;
+
+  &:focus-within {
+    outline: -webkit-focus-ring-color auto 1px;
+    .textareaDetail {
+      border-bottom-color: transparent;
```

Comment:
There's a small spacing caused by the 'rta' class from react-textarea-autocomplete, which wouldn't allow the parent div contained to outline its child elements.